MH4U Dex/QuestDetailsTableViewController.m

```diff
@@ -159,4 +201,14 @@ - (Monster *)monsterForIndexPath:(NSIndexPath *)indexPath
     return self.questMonsters[indexPath.row];
 }
 
+- (Region *)regionWithName:(NSString *)regionName
+{
+    NSFetchRequest *regionRequest = [NSFetchRequest fetchRequestWithEntityName:[Region entityName]];
+    NSPredicate *regionPredicate = [NSPredicate predicateWithFormat:@"%K == %@", RegionAttributes.name, regionName];
+    regionRequest.predicate = regionPredicate;
+    NSError *error = nil;
+    Region *region = (Region *)[[CoreDataController sharedCDController].managedObjectContext executeFetchRequest:regionRequest error:&error].firstObject;
```

Comment:
I would break this into a couple of lines - it seems like you're trying to do too many things in one line, and you're not checking the error you pass in.